src/citrine/resources/project.py

```diff
@@ -199,6 +200,48 @@ def make_private(self,
         })
         return True
 
+    def list_members(self) -> List[User]:
+        """
+        List all of the users in the current project.
+
+        Returns
+        -------
+        ListMemberResponse
+            Dict w/ key 'users' containing an array of User objects
```

Comment:
```suggestion
            Dict with key 'users' containing an array of User objects
```

Reply:
It looks like the method extracts the 'users' field, so this should be "List of users in the current project."